drivers/sensor/bmi270/bmi270_trigger.c

```diff
@@ -84,6 +84,24 @@ static void bmi270_thread_cb(const struct device *dev)
 	}
 }
 
+/* claim the device for PM_DEVICE_RUNTIME if triggers are configured */
+static int handle_pm_device_claim(const struct device *dev)
```

Comment:
What is the purpose of this function? Why not call the PM functions directly?

Reply:
I just moved to stuff into the calling function